packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/TimelineHeaderRow/TimelineCollapser.tsx

```diff
@@ -44,16 +45,16 @@ export default class TimelineCollapser extends React.PureComponent<CollapserProp
     return (
       <div className="TimelineCollapser" ref={this.containerRef}>
         <Tooltip title={getTitle('Expand +1')} getPopupContainer={this.getContainer}>
-          <Icon type="right" onClick={onExpandOne} className="TimelineCollapser--btn-expand" />
+          <RightOutlined onClick={onExpandOne} className="TimelineCollapser--btn-expand" />
         </Tooltip>
         <Tooltip title={getTitle('Collapse +1')} getPopupContainer={this.getContainer}>
-          <Icon type="right" onClick={onCollapseOne} className="TimelineCollapser--btn" />
+          <RightOutlined onClick={onCollapseOne} className="TimelineCollapser--btn" />
         </Tooltip>
         <Tooltip title={getTitle('Expand All')} getPopupContainer={this.getContainer}>
-          <Icon type="double-right" onClick={onExpandAll} className="TimelineCollapser--btn-expand" />
+          <DoubleRightOutlined onClick={onExpandAll} className="TimelineCollapser--btn-expand" />
         </Tooltip>
         <Tooltip title={getTitle('Collapse All')} getPopupContainer={this.getContainer}>
-          <Icon type="double-right" onClick={onCollapseAll} className="TimelineCollapser--btn" />
+          <DoubleRightOutlined onClick={onCollapseAll} className="TimelineCollapser--btn" />
```

Comment:
These changes look odd, do we have the same icons repeated twice? Can you add before/after screenshots?

Reply:
# Before in V3
![before_double-right](https://github.com/jaegertracing/jaeger-ui/assets/89722848/8ea0bd2b-2917-40fa-ae53-35b013d02bd1)
# After in V4
![doublerightOutlined](https://github.com/jaegertracing/jaeger-ui/assets/89722848/fc728de0-57c4-4540-8446-8c61e0ae5016)

For more Information Checkout: 
- [V4 Icons library](https://4x.ant.design/components/icon/)
- [V3 Icons library](https://3x.ant.design/components/icon/)
The below shows the official change in the antdesign icons
![icon_changes](https://github.com/jaegertracing/jaeger-ui/assets/89722848/225f7f23-8ec6-42d7-a62d-615fd46f2998)

Reply:
This wasn't my question, but it's ok, I found the answer - despite using the same icons, every 2nd copy of it is rotated 90 degrees, which makes them look different
![image](https://github.com/jaegertracing/jaeger-ui/assets/3523016/b8bfe1d9-a6b2-4618-a356-2104a0354c68)